astroid/const.py

```diff
@@ -12,7 +12,7 @@
 
 WIN32 = sys.platform == "win32"
 
-IMPLEMENTATION_PYPY = platform.python_implementation() == "PyPy"
+PYPY = platform.python_implementation() == "PyPy"
```

Comment:
Just a thought, can this be done with `sys` as well? I checked the import time of `platform` recently and it's quite a lot for what we are using it for. Are there any possibilities with already imported packages?

Reply:
We should be able to use `sys.implementation.name` instead. Let me do this in a followup. I'll revert the renaming for now to keep the diff smaller. https://docs.python.org/3/library/sys.html#sys.implementation